src/client.rs

```diff
@@ -1,23 +1,24 @@
+//! WebDriver client implementation.
+
 use crate::elements::{Element, Form};
+use crate::error;
 use crate::session::{Cmd, Session, Task};
 use crate::wait::Wait;
-use crate::{error, Locator};
+use crate::wd::{Capabilities, Locator, NewWindowType, WindowHandle};
 use hyper::{client::connect, Method};
 use serde_json::Value as Json;
-use std::convert::TryFrom;
+use std::convert::{TryFrom, TryInto as _};
 use std::future::Future;
 use tokio::sync::{mpsc, oneshot};
-use webdriver::command::{
```

Comment:
Why get rid of these `use`s?

Reply:
Across this file there was a mix: some places use long form with `webdriver::command::` prefix, other ones use short form. I've found it quite confusing. As the later was minority and the imported names weren't much, but in few places only, I've made the style consitent accross this file.